build.sbt

```diff
@@ -215,9 +247,29 @@ def mimaSettings(moduleName: String) = {
   lazy val extraVersions: Set[String] = Set()
 
   Seq(
-    mimaPreviousArtifacts := (mimaVersions(version.value) ++ extraVersions)
-      .filterNot(excludedVersions.contains(_))
-      .map(v => "org.typelevel" %% moduleName % v)
+    mimaPreviousArtifacts := { if(priorTo2_13(scalaVersion.value)) {
+      (mimaVersions(version.value) ++ extraVersions)
+        .filterNot(excludedVersions.contains(_))
+        .map(v => "org.typelevel" %% moduleName % v)
+    } else Set() },
+
+    mimaBinaryIssueFilters ++= {
+      import com.typesafe.tools.mima.core._
+      import com.typesafe.tools.mima.core.ProblemFilters._
+      //Only sealed abstract classes that provide implicit instances to companion objects are allowed here, since they don't affect usage outside of the file.
```

Comment:
This would be a good option to add to mima: ignore changes when the or trait is sealed.

Reply:
I created a placeholder issue at the mima repo, but it needs fleshing out to become actionable. https://github.com/lightbend/migration-manager/issues/237